### ISSUE TITLE
An Introduction to Accessibility in React

### ISSUE THREAD

**Issue by jsscclr** [reactsydney/talks#72]
## Talk Date
7 May 2018 (or whenever really!)

## Talk Title
An Introduction to Accessibility in React

## Length
~45 minutes

## Twitter-sized Overview (140 characters)
Your site is inaccessible and it's your fault and you should feel bad. Just kidding! ...Sort of. It's not React's fault your site is inaccessible, but it's not exactly _helping_ you either. What can we do about it?

## Detailed Overview
We all (hopefully) know that accessibility is important. There are an endless number of resources available telling you about how to make your site accessible, but a lot less telling you how to make your React app accessible.

The React docs state `React fully supports building accessible websites, often by using standard HTML techniques.` And that's true! But whilst React doesn't prevent you from making your site accessible, it (and single page applications in general) _can_ make some things a bit harder. It's not all bad news, React can also make some things a lot easier!

This talk will cover a brief introduction to accessibility concepts such as WCAG compliance and dealing with screen readers, as well as covering low hanging fruit (yes, please add those alt attributes to your images!). The main part of this talk is covering issues you're likely to run into when using React, such as handling navigation, updating document meta information, tab order and focus management in general.

Don't care about accessibility? Okay, you probably _should_ feel bad, but there's something for you as well: turns out those shiny new React features like Fragments and Portals are super helpful in the quest to build accessible websites!

**Comment by jesstelford** [comment]
Hey @jsscclr we'd love to have this talk at React Sydney - it's an important topic that everyone should be implementing every day!

Are you able to give this talk at the May 7th meetup?

**Comment by jsscclr** [comment]
Would love to :D

**Comment by jesstelford** [comment]
Hey Jess, checking in - anything I can help with to get you prepared for the event on Monday?

**Comment by jsscclr** [comment]
All good from me :) Thanks!

**Comment by jesstelford** [comment]
Really looking forward to this talk! We'll have you up as the third speaker for tonight.

**Comment by jesstelford** [comment]
A most excellent talk! Now available on YouTube: https://www.youtube.com/watch?v=XXI-Ydae0qw
